pages/linux/amap.md

```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org>.
+
+- Scan port 80 on a target host and print ASCII banner response, omit closed ports:
+
+`amap -bqv {{ip_or_hostname}} {{80}}`
+
+- Scan port 1-1024 and capture banners:
+
+`amap -bqv {{ip_or_hostname}} {{1-1024}} -B`
```

Comment:
```suggestion
- Scan a specific port and capture banners:

`amap -bqv {{ip|hostname}} {{1..1024}} -B`
```

Almost the same issue as above.
Warning: mnemonics can't be suggested for options as parsing man page failed and possibly several short options differing just with case detected.
Optional fix: `specific` may sound better.
Optional fix: `..` can be used for ranges, our Emacs extension accepts just such syntax as correct (standardization).